components/addTask.vue

```diff
@@ -47,15 +49,47 @@ import { defineComponent } from '@nuxtjs/composition-api'
 
 export default defineComponent({
   emits: ['newTask'],
+  data(){
+    return{
+      inputText:""
+    }
+  },
   methods: {
-    addTask() {
+    async addTask() {
       /**
        * @todo Complete this function.
        * @todo 1. Send the request to add the task to the backend server.
        * @todo 2. Add the task in the dom.
        * @hint use emit to make a event that parent can observe
        */
+      console.log("debyg");
+      console.log(this.inputText);
+
+      if(this.inputText=='')
+      {
+      this.$toast.error('Cannot Add empty Todo');
```

Comment:
Fix indentation and capitalize the error message properly. Ditto below

Reply:
Yeah, using prettier is fine. However, a habit of writing clean code goes a long way.